appserver/tests/paas/lazy-shared-service-test/pom.xml

```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
```

Comment:
I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence